bands/site_sections/band_admin.py

```diff
@@ -3,12 +3,51 @@
 
 @all_renderable(c.BANDS)
 class Root:
+
+    def _required_message(self, params, fields):
```

Comment:
Ugh, yes, this really should be pulled into the main plugin. I didn't put it there out of sheer laziness. Moving this to the main plugin just seemed like it would be too arduous. I'd have to open a new pull request in a separate repo, and then wait for that to be approved before the tests would pass in this pull request.

This is another big reason why I want to move away from the multiple plugin architecture.

Also, this method is kinda weird and a little specific to this scenario, because not all those fields belong to the same model (`name` belongs to `Group` and the rest belong to `Attendee`).